condor_watch_q.py

```diff
@@ -216,6 +217,26 @@ def parse_args():
     return args
 
 
+def parse_unknown_args(unknown):
+    for unknown_arg in unknown:
+        print("Unknown command.", end=" ")
+        if unknown_arg.isdigit():
+            print("Did you mean condor_watch_q -clusters {}?".format(unknown_arg))
+        elif "-totals" in unknown_arg:
+            print("Did you mean condor_watch_q -no-table?")
+        elif "-userlog" in unknown_arg:
+            print("Did you mean condor_watch_q -files FILE [FILE ...]?")
+        elif "-nobatch" in unknown_arg:
+            print("Defaults to batch, try condor_watch_q -groupby {batch,log,cluster}.")
+        elif "-" in unknown_arg:
+            print(
+                "condor_watch_q does not support the following functionality yet."
+            )
```

Comment:
I'm not sure I like this. I assume this is because argparse is no longer throwing errors when you pass unknown options. I'd prefer if we ended up sending the unknown args back to the parser to produce the "real" error messages after printing out the "did you mean...?" helpers. Might be tricky to get the message ordering right... maybe something like this pseudocode?
```
try:
    parse all args
except <whatever error is thrown for unknown args>:
    parse known args
    create help messages
    print argparse error
    print help messages
    exit(1)
```